src/surge-fx/KnobSource.h

```diff
@@ -36,4 +40,298 @@ struct ConcreteCM : sst::jucegui::data::ContinuousModulatable
     float getModulationValuePM1() const override { return mv; }
     void setModulationValuePM1(const float &f) override { mv = f; }
     bool isModulationBipolar() const override { return isBipolar(); } // sure why not
+};
+
+struct StyleSheetBuiltInImp : public sst::jucegui::style::StyleSheet
```

Comment:
Ooooh we shouldn't need all this code. You should be able to start with the Dark and customize colors from there.

Reply:
https://github.com/surge-synthesizer/shortcircuit-xt/blob/main/src-ui/theme/ThemeApplier.cpp here's how SC does it

Reply:
example makes sense to me. Will update tmrw.